### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by ynahum**

### CODE REVIEW

File: Lecture12.tex [hunk 1]
```diff
@@ -204,7 +204,7 @@ \subsection{Variance Reduction}
 where $\Psi_t$ can be either one of the following terms:
 \begin{enumerate}
     \item Total reward, $\sum_{t=0}^\infty r(x_t,u_t) - b(x_t)$, where $b$ is a state-dependent baseline
-    \item Future reward following action $u_t$, $\sum_{t'=t}^\infty r(x_t,u_t) - b(x_t)$
+    \item Future reward following action $u_{t'}$, $\sum_{t'=t}^\infty r(x_{t'},u_{t'}) - b(x_{t'})$
```
Comment (avivt) [inline Lecture12.tex]: the first term should still be u_t.

File: Lecture5.tex [hunk 2]
```diff
@@ -312,7 +312,7 @@ \section{Proof of Bellman's Optimality Equation}
...
   \item By definition of  ${\pi ^*}$ we have
 \[T_{}^{\pi ^*}(V_{}^*) = T_{}^*(V_{}^*) = V_{}^*,\]
-where the last equality follows from part 1. Thus the optimal value function satisfied the equation $T_{}^{\pi ^*}(V_{}^*) = V_{}^*$. But we already know (from Prop. \ref{prop:FP_VI}) that $V_{}^{\pi ^*}$ is the unique solution of that equation, hence $V_{}^{\pi ^*} = V_{}^*$. This implies that ${\pi ^*}$ achieves the optimal value (for any initial state), and is therefore an optimal policy as stated.
+where the last equality follows from part 1. Thus the optimal value function satisfied the equation $T_{}^{\pi ^*}(V_{}^*) = V_{}^*$. But we already know (from Theorems. \ref{thm:_BANACH} and \ref{thm:_CONTRACT}) that $V_{}^{\pi ^*}$ is the unique solution of that equation, hence $V_{}^{\pi ^*} = V_{}^*$. This implies that ${\pi ^*}$ achieves the optimal value (for any initial state), and is therefore an optimal policy as stated.
```
Comment (avivt) [inline Lecture5.tex]: no dot after Theorems

File: Lecture12.tex (hunk 1 above)
Comment (ynahum) [reply Lecture12.tex]: fixed in a new patch

File: Lecture5.tex (hunk 2 above)
Comment (ynahum) [reply Lecture5.tex]: fixed in a new patch

**Comment by avivt** [comment]
Thank you!

**Pull request merged**